Add new influxdb/host shortcode and implement it in 3.0 docs (#5077)
influxdata/docs-v2: CONTRIBUTING.md

- **method**: HTTP request method (get, post, patch, put, or delete)
- **endpoint**: API endpoint
- **api-ref**: Link the endpoint to a specific place in the API documentation

> - **influxdb_host**: Specify which InfluxDB version host to use

Reviewer: ```suggestion
- **influxdb_host**: Specify which InfluxDB product host to use
```
Product or platform instead of version?

> - **influxdb_host**: Specify which InfluxDB version host to use

Reviewer: ```suggestion
- **influxdb_host**: Specify which InfluxDB product host to use
```
Product or platform instead of version?

  _if the `endpoint` contains the `influxdb/host` shortcode_.

>   Uses the current InfluxDB version as default.

Reviewer: ```suggestion
  Uses the current InfluxDB product as default.
```
Product or platform instead of version?

>   Supports the following values:

Reviewer: ```suggestion
  Supports the following product values:
```

```
~~~

### Automatically populate InfluxDB host placeholder
The InfluxDB host placeholder that gets replaced by custom domains differs
between each InfluxDB product/version.
Use the `influxdb/host` shortcode to automatically render the correct

> host placeholder value for the current version. You can also pass a single

Reviewer: ```suggestion
host placeholder value for the current product. You can also pass a single
```

> argument to specify a specific version of InfluxDB to use.

Reviewer: ```suggestion
argument to specify a specific InfluxDB product to use.
```
